packages/flakybot/src/flakybot.ts

```diff
@@ -120,6 +146,14 @@ export function flakybot(app: Probot) {
     const commit = context.payload.commit || '[TODO: set commit]';
     const buildURL = context.payload.buildURL || '[TODO: set buildURL]';
 
+    const config = (await getConfig<Config>(
+      context,
+      owner,
+      repo,
+      CONFIG_FILENAME,
+      DEFAULT_CONFIG
+    )) as Config; // This is safe because we pass the default config.
```

Comment:
If you do the multiple type heads, you could make this call without the explicit cast.

Reply:
right, will do